### ISSUE TITLE
no-unnecessay-collection does not support spreading

### ISSUE THREAD

**Issue by reduckted** [JasonWeinzierl/eslint-plugin-rxjs-x#288]
Spreading an array into an observable creator triggers a false positive for `no-unnecessay-collection`
```ts
merge(...observables)
```
